Base/QTCore/qSlicerCoreApplication.cxx

```diff
@@ -1746,25 +1746,28 @@ QString qSlicerCoreApplication::libraries()const
 //-----------------------------------------------------------------------------
 QString qSlicerCoreApplication::copyrights()const
 {
-  QString copyrightsText(
+  QString copyrightsText(QString(
     "<table align=\"center\" border=\"0\" width=\"80%\"><tr>"
-    "<td align=\"center\"><a href=\"https://slicer.readthedocs.io/en/latest/user_guide/about.html#license\">Licensing Information</a></td>"
-    "<td align=\"center\"><a href=\"https://slicer.org/\">Website</a></td>"
-    "<td align=\"center\"><a href=\"https://slicer.readthedocs.io/en/latest/user_guide/about.html#acknowledgments\">Acknowledgments</a></td>"
-    "</tr></table>");
+    "<td align=\"center\"><a href=\"https://slicer.readthedocs.io/en/latest/user_guide/about.html#license\">%1</a></td>"
+    "<td align=\"center\"><a href=\"https://slicer.org/\">%2</a></td>"
+    "<td align=\"center\"><a href=\"https://slicer.readthedocs.io/en/latest/user_guide/about.html#acknowledgments\">%3</a></td>"
+    "</tr></table>")
+    .arg(tr("Licensing Information"))
+    .arg(tr("Website"))
+    .arg(tr("Acknowledgments")));
   return copyrightsText;
 }
 //-----------------------------------------------------------------------------
 QString qSlicerCoreApplication::acknowledgment()const
 {
   QString acknowledgmentText(
-    "Slicer is NOT an FDA approved medical device.<br /><br />"
-    "Supported by: NA-MIC, NAC, BIRN, NCIGT and the Slicer Community.<br /><br />"
-    "Special thanks to the NIH and our other supporters.<br /><br />"
-    "This work is part of the National Alliance for Medical Image Computing "
-    "(NA-MIC), funded by the National Institutes of Health through the NIH "
-    "Roadmap for Medical Research, Grant U54 EB005149. Information on the "
-    "National Centers for Biomedical Computing can be obtained from "
+    tr("Slicer is NOT an FDA approved medical device.") + "<br /><br />" +
+    tr("Supported by: NA-MIC, NAC, BIRN, NCIGT and the Slicer Community.") + "<br /><br />" +
+    tr("Special thanks to the NIH and our other supporters.") + "<br /><br />" +
+    tr("This work is part of the National Alliance for Medical Image Computing ") +
+    tr("(NA-MIC), funded by the National Institutes of Health through the NIH ") +
```

Comment:
Each translatable string must be a complete sentence. Please update this accordingly and also review all other strings in this pull request and fix as needed.

Reply:
Thank you for your feedback. 

Translatable strings are now grouped as complete sentences wherever it's needed.

Regards.